src/aiortc/contrib/media.py

```diff
@@ -7,7 +7,7 @@
 import sys
 from typing import Dict, Optional, Set
 import numpy as np
-
+import concurrent.futures
```

Comment:
wouldn't get rid of current empty lines, just add a new line rather than insert text in between

Reply:
Where so you mean to add a new line?